static/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+    "realm": "fcf0kpqr",
+    "auth-server-url": "https://dev.oidc.gov.bc.ca/auth/" ,
+    "ssl-required": "external",
+    "resource": "ppr-web",
+    "public-client": true,
+    "confidential-port": 0
+}
```

Comment:
Do you need to keep these keycloak files? I think the dev proxy gets it from ppr-dev right now anyways and we will have it in a config mounted in openshift for dev/test/prod

Reply:
We are currently generating a static site that is deployed to firebase. We run npm run generate, then deploy with a firebase api. So we need all configuration before generating. With this deployment we need to figure out how to update keycloak.json by environment as part of the generate step.